src/relentless/simulate/lammps.py

```diff
@@ -257,6 +281,27 @@ def pair_map(sim, pair):
                         fw.write(
                             "{idx} {r} {u} {f}\n".format(idx=idx, r=r_, u=u_, f=f_)
                         )
+
+                # write bond potentials into the file
+                if has_bonds:
+                    fw.write("# LAMMPS tabulated bond potentials\n")
```

Comment:
Just to confirm: LAMMPS is fine with both the pair and bond potential being in the same file?

Reply:
My understanding is that the bond_coeff command takes the file name argument and then for a specific entry in that file just like pair. The LAMMPS examples provided being: 
```
bond_style table linear 1000
bond_coeff 1 file.table ENTRY1
```
and 
```
pair_style table linear 1000
pair_style table linear 1000 pppm
pair_style table bitmap 12
pair_coeff * 3 morse.table ENTRY1
pair_coeff * 3 morse.table ENTRY1 7.0
```
I've not had any issues with them being in the same file, but if you think that could cause a problem, we can change it!

Reply:
If it works, I'm good with it! It will make it cleaner to scrub this file later if we want to (see #217)